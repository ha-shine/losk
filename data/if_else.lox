// If statement
if (10 > 5) {
    print "10 is greater than 5";
}

// If statement with else branch
if (5 < 3) {
    print "5 is less than 3";
} else {
    print "5 is greater than 3";
}