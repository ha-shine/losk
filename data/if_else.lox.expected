10 is greater than 5
5 is greater than 3
